JASP-Engine/JASP/R/descriptives.R

```diff
@@ -65,6 +64,7 @@ Descriptives <- function(jaspResults, dataset, options, state=NULL)
       jaspResults[["frequenciesHeading"]] <- createJaspHtml("Frequencies", "h1")
       jaspResults[["frequenciesHeading"]]$copyDependenciesFromJaspObject(jaspResults[["tables"]])
       jaspResults[["frequenciesHeading"]]$dependOnOptions("variables")
+      jaspResults[["frequenciesHeading"]]$position <- 2
```

Comment:
@koenderks

Reply:
Pretty sure I was the one who messed that up

Reply:
Don't worry about this, there's a three-way merge thingie going on with Koen also working on this. After his work is merged, I'll get someone else to further smoothen this out.